src/components/CioQuiz/index.tsx

```diff
@@ -56,6 +58,7 @@ export default function CioQuiz(props: IQuizProps) {
     getResetQuizButtonProps,
     getSelectInputProps,
     customClickItemCallback: !!resultsPageOptions?.onQuizResultClick,
+    customAddToFavoritesCallback: !!resultsPageOptions?.onAddToFavoritesClick,
```

Comment:
Why add `customAddToFavoritesCallback` to the customer facing interface? I'm not opposed to using it just why return it in the hook? I think we can remove it along with `customClickItemCallback`

Reply:
Ignore this please